src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -417,6 +433,15 @@ private void OnStartPageLoaded(object sender, RoutedEventArgs e)
 
             var id = Wpf.Interfaces.ResourceNames.StartPage.Image;
             StartPageLogo.Source = dynamoViewModel.BrandingResourceProvider.GetImageSource(id);
+
+            if (dynamoViewModel.Model.PreferenceSettings.IsFirstRun)
+            {
+                dynamoViewModel.Model.PreferenceSettings.IsFirstRun = false;
```

Comment:
Hi @Sancerio, it is not the job of `StartPageView` to set `PreferenceSettings.IsFirstRun = false`. `UsageReportingManager.CheckIsFirstRun` will do that, since it is a very important component.